news/2022-03-22-community-contributors-elite-nominators.md

```diff
@@ -0,0 +1,127 @@
+A long-standing member of the osu!catch Beatmap Nominator community, [**Nokashi**](https://osu.ppy.sh/users/5431196) returned to prominence in the scene during late 2020 and has made his contributions well and truly felt since.
+
+Though he wins no awards for an extraordinary quantity of nominations, the sheer quality of his feedback demonstrates an unyielding dedication towards detail and nuance, showing that less truly is sometimes much, much more. His efforts to assist newer mappers attempting to acclimate to the unusual creation environment in osu!catch have not gone unnoticed either, nor have his efforts in other modding-focused projects such as his management of the Community Mentorship program for his game mode.
+
+## osu!mania
+
+[![](/wiki/shared/news/2022-03-22-community-contributors-elite-nominators/Quenlla.jpg)](https://osu.ppy.sh/users/4725379)
+
+Powering into the osu!mania scene during 2021, it should come as no surprise to anyone that [**Quenlla**](https://osu.ppy.sh/users/4725379) was easily the uncontested most active Beatmap Nominator. Initially renowned for pushing a lot of variety in the 4K scene, she later pivoted towards helping newer mappers get their work seen and encouraging a goodly number of osu!mania tournament maps to take their rightful place in Ranked.
+
+Quenlla has played a critical role in recent Ranking Criteria revisions and has had a large presence both Project Loved and the osu!mania World Cups, topping it all off with an introduction to the Nomination Assessment Team earlier in 2022.
+
+[![](/wiki/shared/news/2022-03-22-community-contributors-elite-nominators/guden.jpg)](https://osu.ppy.sh/users/11626065)
+
+Noted across the osu!mania community for his open-mindedness when approaching beatmaps of all kinds, [**guden**](https://osu.ppy.sh/users/11626065)'s mixture of modding and curation in the 4K scene has been directly responsible for some of the most innovative and interesting additions to Ranked in recent years, with picks like [Ibitbt](https://osu.ppy.sh/beatmapsets/1287682) and [Xeroa](https://osu.ppy.sh/beatmapsets/1305067) coming to mind.
+
+Splitting his attention between numerous other projects in the scene, guden is especially remarkable for the extremely high quality modding he produces, all while playing a large part in many tournaments as a pooler and mapper alike.
+
+[![](/wiki/shared/news/2022-03-22-community-contributors-elite-nominators/stan.jpg)](https://osu.ppy.sh/users/1653229)
+
+A dedicated Beatmap Nominator since 2018, [**\_Stan**](https://osu.ppy.sh/users/1653229) has had a noted passion for pushing new beatmaps into the 7K scene, ranging from pleasantly simple to some of the hardest beatmaps ever seen to date. 2021 was a continuation of this long-standing tradition for him, powering through nearly thirty nominations with some extremely thorough and detailed modding that have showcased not only his dedication towards improving 7K overall, but his experience with the mode as a whole.
```

Comment:
redundant
```suggestion
A dedicated Beatmap Nominator since 2018, [**\_Stan**](https://osu.ppy.sh/users/1653229) has had a noted passion for pushing new beatmaps into the 7K scene, ranging from pleasantly simple to some of the hardest beatmaps seen to date. 2021 was a continuation of this long-standing tradition for him, powering through nearly thirty nominations with some extremely thorough and detailed modding that have showcased not only his dedication towards improving 7K overall, but his experience with the mode as a whole.
```